src/main/resources/com/cloudbees/jenkins/support/Messages.properties

```diff
@@ -28,7 +28,11 @@ SupportAction_DefaultActionBlurb=\
   In order to assist the Jenkins community to provide you with a response to your issues please consider generating \
   a bundle of the commonly requested information and consider attaching this bundle to the corresponding issue in \
   the Jenkins community issue tracker. Be mindful that attachments to the Jenkins community issue tracker are public \
-  so be extra careful not to include sensitive information.
+  so be extra careful not to include sensitive information. Note that certain metadata can be automatically \
+  anonymized when support bundle anonymization is enabled through the global configuration settings. \
+  When anonymization is enabled, the following information is replaced with automatically generated anonymized names: \
+  nodes, computers, labels, users, items, views, and IP addresses. Original names can be looked up via the \
```

Comment:
I might reword this to `the names of nodes, computers, labels, users, items (including jobs), and views, and IP addresses`.